docs/source/onnx.md

```diff
@@ -88,3 +88,29 @@ for tracking purposes -->
 .. py:module:: torch.onnx.symbolic_opset9
 .. py:module:: torch.onnx.utils
 ```
+
+Python API
+----------
+
+.. automodule:: torch.onnx
+
+Functions
+^^^^^^^^^
+
+.. autofunction:: export
+    :noindex:
+.. autofunction:: register_custom_op_symbolic
+.. autofunction:: unregister_custom_op_symbolic
+.. autofunction:: select_model_mode_for_export
```

Comment:
These functions are obsolete. We can move them to the section above which will store a reference but will not display them